apps/src/code-studio/pd/application/teacher/TeacherApplication.jsx

```diff
@@ -44,16 +44,16 @@ const TeacherApplication = props => {
     accountEmail,
     userId,
     savedStatus,
-    schoolId
+    schoolId,
+    storage
```

Comment:
I feel like this is a lot of work to just use FakeStorage for testing ... we have to make a prop here, set a default value, and (I think) do the same thing in FormController to ensure nothing gets lost. Is it worth it? Not sure

Reply:
not sure if this answers the question, but it's definitely worth stubbing out window.sessionStorage to prevent leaking state across tests. the way in which you're enabling this is a legit pattern to use. I would go with it if you cannot find a simpler one (e.g. stubbing window directly).